speedy-recovery-client/src/components/secured/shared/Header.js

```diff
@@ -78,6 +90,47 @@ class Header extends Component {
       </header>
     );
   }
+
+  componentDidMount() {
+    this._isMounted = true;
```

Comment:
I can still see the warnings. Also using isMounted is not a good solution anymore:
https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html,
https://github.com/facebook/react/issues/5465#issuecomment-157888325